pom.xml

```diff
@@ -47,7 +47,7 @@
     <jenkins.host.address />
     <slaveAgentPort />
     <java.level>8</java.level>
-    <jenkins.version>2.190.1</jenkins.version>
+    <jenkins.version>2.217</jenkins.version>
```

Comment:
Could be made compatible with 2.204.x if desired by using a bit of reflection.

Reply:
Is this necessary to have Websockets enabled? This feature would help us in our setup a lot but we are on LTS so we cannot upgrade the plugin to version 1.25. 😢

Reply:
Wait for the next LTS.

Reply:
2.222.1 should be available within a month or so.